examples/SPDXXMLExample-v2.0.xml

```diff
@@ -140,53 +149,282 @@ THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR IMPLIED WAR
-<relationships>
-<Relationship>
-	  <spdxElementId>SPDXRef-DOCUMENT</spdxElementId>
-	  <relationshipType>CONTAINS</relationshipType>
-      <comment></comment>
-      <relatedSpdxElementId>SPDXRef-Package</relatedSpdxElementId>
-</Relationship>
-	  <spdxElementId>SPDXRef-DOCUMENT</spdxElementId>
-      <relationshipType>DESCRIBES</relationshipType>
-      <comment></comment>
-      <relatedSpdxElementId>SPDXRef-File</relatedSpdxElementId>
-</Relationship>
-<Relationship>
-	  <spdxElementId>SPDXRef-DOCUMENT</spdxElementId>
-      <relationshipType>COPY_OF</relationshipType>
-      <comment></comment>
-      <relatedSpdxElementId>DocumentRef-spdx-tool-1.2:SPDXRef-ToolsElement</relatedSpdxElementId>
-</Relationship>
-<Relationship>
-	  <spdxElementId>SPDXRef-DOCUMENT</spdxElementId>
-      <relationshipType>DESCRIBES</relationshipType>
-      <comment></comment>
-      <relatedSpdxElementId>SPDXRef-Package</relatedSpdxElementId> 
-</Relationship><Relationship>
-	  <spdxElementId>SPDXRef-CommonsLangSrc</spdxElementId>
-      <relationshipType>DYNAMIC_LINK</relationshipType>
-      <comment></comment>
-      <relatedSpdxElementId>SPDXRef-Package</relatedSpdxElementId>
-</Relationship>
-<Relationship>
-	  <spdxElementId>SPDXRef-JenaLib</spdxElementId>
-      <relationshipType>DYNAMIC_LINK</relationshipType>
-      <comment></comment>
-      <relatedSpdxElementId>SPDXRef-Package</relatedSpdxElementId>
-</Relationship>
-<Relationship>
-	  <spdxElementId>SPDXRef-Package</spdxElementId>
-	  <relationshipType>DYNAMIC_LINK</relationshipType>
-	  <comment></comment>
-	  <relatedSpdxElementId>SPDXRef-Saxon</relatedSpdxElementId>
-</Relationship>
-<Relationship>
-	  <spdxElementId>SPDXRef-Package</spdxElementId>
-	  <relationshipType>CONTAINS</relationshipType>
-	  <comment></comment>
-	  <relatedSpdxElementId>SPDXRef-JenaLib</relatedSpdxElementId>
-</Relationship>
-</relationships>
-</Document></SpdxDocument>
\ No newline at end of file
+*/</extractedText>
+    <licenseId>LicenseRef-1</licenseId>
+  </hasExtractedLicensingInfo>
+  <annotation>
+    <annotationDate>2011-03-13T00:00:00Z</annotationDate>
+    <annotationType>REVIEW</annotationType>
+    <annotator>Person: Suzanne Reviewer</annotator>
+    <comment>Another example reviewer.</comment>
+  </annotation>
+  <annotation>
+    <annotationDate>2010-02-10T00:00:00Z</annotationDate>
+    <annotationType>REVIEW</annotationType>
+    <annotator>Person: Joe Reviewer</annotator>
+    <comment>This is just an example.  Some of the non-standard licenses look like they are actually BSD 3 clause licenses</comment>
+  </annotation>
+  <annotation>
+    <annotationDate>2010-01-29T18:30:22Z</annotationDate>
+    <annotationType>OTHER</annotationType>
+    <annotator>Person: Jane Doe ()</annotator>
+    <comment>Document level annotation</comment>
+  </annotation>
+  <documentDescribes>SPDXRef-Package</documentDescribes>
+  <documentDescribes>SPDXRef-File</documentDescribes>
+  <packages>
```

Comment:
Is it intentional that the tag for a single Package would be `<packages>`? I would think that it would be `<package>`, and/or that perhaps there would be one enclosing `<packages>` tag that would encompass all of the individual `<package>` tags. But I am absolutely not an XML expert in any sense  :)

Reply:
I have the same question - this is one of those structural differences between JSON/YAML and XML/RDF/XML formats.